Documentation/DocuBlocks/Rest/Replication/put_api_replication_revisions_documents.md

```diff
@@ -0,0 +1,68 @@
+
+@startDocuBlock put_api_replication_revisions_documents
+@brief retrieves documents by revision
+
+@RESTHEADER{GET /_api/replication/revisions/documents, Return documents by revision,handleCommandRevisionDocuments}
+
+@RESTQUERYPARAMETERS
+
+@RESTQUERYPARAM{collection,string,required}
+The name or id of the collection to query.
+
+@RESTQUERYPARAM{batchId,number,required}
+The id of the snapshot to use
+
+@RESTDESCRIPTION
+Returns documents by revision
+
+The body of the request should be JSON/VelocyPack and should consist of an
+array of string-encoded revision IDs:
+
+```
+[
+  <String, revision>,
+  <String, revision>,
+  ...
+  <String, revision>
+]
+```
+
+In particular, the revisions should be sorted in ascending order of their
```

Comment:
In hindsight, we should really have used this alphabet for the encoding: "-0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz" then lexicographic string order would be the same as decoded value order. Well, too late. And the "official" base64 committed the same blunder in a different way. LOL.